src/test/controllers/company.details.controller.spec.unit.ts

```diff
@@ -47,6 +49,8 @@ mockGetRequest.mockReturnValue(
  describe("company.details.controller tests", () => {
 
    beforeEach(() => {
+    mockMiddlewares.mockCsrfProtectionMiddleware.mockClear();
```

Comment:
Indentation slightly out.

Reply:
my configuration only indents the lines I have touched

Reply:
ok, but the new line is one character to the left of the rest of the block

Reply:
done